elements/lisk-p2p/src/peer/base.ts

```diff
@@ -201,7 +209,28 @@ export class Peer extends EventEmitter {
 			this.emit(EVENT_REQUEST_RECEIVED, request);
 		};
 
-		this._handleWSMessage = () => {
+		this._handleWSMessage = (message: any) => {
```

Comment:
Please cover this with unit tests.

Reply:
It's covered in the integration test as we require 2 nodes to test it properly